src/traits.rs

```diff
@@ -484,6 +467,69 @@ pub trait RemLimb: Sized {
     fn rem_limb_with_reciprocal(&self, reciprocal: &Reciprocal) -> Limb;
 }
 
+/// Bit counting and bit operations.
+pub trait BitOperations {
```

Comment:
I think it might be a bit more idiomatic to call this `BitOps`, but this is fine

Reply:
I'll rename it.

Reply:
Renamed